### ISSUE TITLE
Tooling: Detect and show a warning if an `NSLocalizedString`'s value has been changed but not its key

### ISSUE THREAD

**Issue by twstokes** [wordpress-mobile/WordPress-iOS#20029]
If a keyed `NSLocalizedString`, e.g.:

```
NSLocalizedString("some.string.key", value: "Hello, wordl!", comment: "Comment to translators")
```

is changed to:

```
NSLocalizedString("some.string.key", value: "Hello, world!", comment: "Comment to translators")
```

there's nothing to tell the developer that this won't spawn a re-translation of the string because the key didn't change.

Could we detect this and show a warning / reminder of what steps should be taken to update the translation?

**Comment by mokagio** [comment]
I think this issue can be closed based on what has been [reported in #20009](https://github.com/wordpress-mobile/WordPress-iOS/pull/20009#issuecomment-1408000210): Changing the `value` of a `-LocalizedString` will result in the `string` definition updating the next time we run `bundle exec fastlane generate_strings_file_for_glotpress` 👍

**Comment by twstokes** [comment]
Awesome! That's great to hear. Thanks @mokagio. 🙇
